### PR TITLE
HTML Escape

### PR DESCRIPTION
**Pull request by gambol99**
They've updated the version of FreeMaker which no longer requires the html escaping

### CODE REVIEW

**Comment by gambol99** [comment]
Can one of you guy's @daniel-ac-martin / @paulwaitehomeoffice take a gander please :-) .. it was required for an update to keycloak 3.4.3 .. Note, never used freemaker before so going on the google results here :-)

File: govuk-social-providers/login/login.ftl [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<#ftl output_format='HTML'>
```
Comment (pauldwaite) [inline govuk-social-providers/login/login.ftl]: Do we need this line? It isn’t used in any of Keycloak’s own templates (they only use `<#ftl output_format="plainText">` for their e-mail templates), and we haven’t used it anywhere else.

Comment (gambol99) [reply govuk-social-providers/login/login.ftl]: thank you kindly ... was let by accident .. .. removed

**Comment by paulwaitehomeoffice** [comment]
Sure. I don’t think we need the <#ftl output_format="HTML"> line, but aside from that, it looks great.


Do you know where auto-escaping got turned on for the Keycloak Freemarker templates? Is that part of Keycloak? Or is it a change we made?

________________________________

**Comment by daniel-ac-martin** [comment]
Just tried setting the HTML title to
```
<a id="proposition-name" href=".">Keycloak</a>
```
That seems to come through escaped which isn't what we want.

**Comment by daniel-ac-martin** [comment]
Hang on, I'll push up a commit...

**Comment by paulwaitehomeoffice** [comment]
👍 I concur

**Pull request merged**